spec/acceptance/99_facts_spec.rb

```diff
@@ -3,6 +3,15 @@
 describe 'sensu::backend class', if: ['base','full'].include?(RSpec.configuration.sensu_mode) do
   backend = hosts_as('sensu-backend')[0]
   agent = hosts_as('sensu-agent')[0]
+  let(:facter_command) do
```

Comment:
This code is getting duplicated. To DRY things up, could it be moved to the spec_helper?

Reply:
Not really. It's best to define variables in rspec using `let` to ensure proper test isolation and not a great way to do that in central place that I know of. It's also slightly differen logic for Windows tests than other tests so not much to DRY up.